### PR TITLE
ffmuc-mesh-vpn-wireguard: Cleanup Code

### PR DESCRIPTION
**Pull request by T0biii**
# Cleanup Code

- Move code Up for early exiting
- remove unnessery code
- aline v1 code with v2 code (same order)
- remove/add some spaces
- add Info About exiting to loggers (if the scritps exist afterwards)
- move echo to logger (or should it be a echo?)

### CODE REVIEW

File: ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink [hunk 1]
```diff
@@ -4,13 +4,21 @@
...
+if [[ "${mesh_vpn_enabled}" == "0" ]]; then
+	# Stop the script if mesh_vpn is disabled
+	exit 0
+fi
+
 if { set -C; true 2>/dev/null >/var/lock/checkuplink.lock; }; then
 	trap "rm -f /var/lock/checkuplink.lock" EXIT
 else
-	echo "Lock file exists... exiting"
+	logger -p err -t checkuplink "Lock file exists... exiting."
```
Comment (grische) [inline ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink:17]: This isn't really an error, it probably is better to be a notice. This is triggered when the code below takes longer than the rerun from the cronjob. Happens regularly on slow connections.

File: ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink [hunk 2]
```diff
@@ -182,34 +187,24 @@ is_connected() {
 	fi
 	return 1 # false
 }
+# end function section
```
Comment (grische) [inline ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink:190]: I'm not sure how helpful this comment here is. What about adding a start section comment for the actual code instead?

File: ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink (hunk 1 above)
Comment (T0biii) [reply ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink:17]: sounds good to me

File: ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink (hunk 2 above)
Comment (T0biii) [reply ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink:190]: then i also remove the start info here: https://github.com/freifunk-gluon/community-packages/pull/107/files#diff-5bc274003ba90958a75ae0d119e24d89fb0d4acab0bf03ac24797002750e22f3R21

and add something like "# start main logic"?

Comment (grische) [reply ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink:190]: Sure, if you would like that.

Comment (T0biii) [reply ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink]: something like this https://github.com/freifunk-gluon/community-packages/pull/107/commits/c740b1adb5d7f12b842856ebe382831d87caa448 ?

Comment (grische) [reply ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink]: Looks fine to me.

**Review by maurerle — APPROVED** [review]

**Pull request merged**